### ISSUE TITLE
Rebranding could lead to confusion and bad UX!

### ISSUE THREAD

**Issue by Majoramari** [Redot-Engine/redot-engine#688]
### Tested versions

[83d6573](https://github.com/Redot-Engine/redot-engine/commit/83d65738b06986e5aa0943cae01e357133e04a75)

### System information

Non OS related

### Issue description

As we know, the node system relies on colors. After the rebranding, there’s some confusion with the already existing nodes, especially those in red and blue.

Additionally, some icons have a low accessibility rate in the new Redot theme.

My suggestion is to choose distinct colors, fonts, and possibly add logo outlines. Alternatively, you could make the theme monochromatic, using only dark gray, to avoid confusion.

### Steps to reproduce

Non related

### Minimal reproduction project (MRP)

None related

**Comment by RadenTheFolf** [comment]
This is a great suggestion to keep in mind. Its important to maintain accessibility in the engine.

**Comment by SkogiB** [comment]
There was one version someone had that boosted the saturation of the icons a lot and I really liked it for more easily distinguishing icons at a glance
![image](https://github.com/user-attachments/assets/d2d50f8b-ad70-4063-bdf0-ec62e4d64c73)

**Comment by CaCTuCaTu4ECKuu** [comment]
I also not sure if It should have its own issue, but there are objective researches of how colour influences productivity and well-being and red color in particular is not optimal in that regard. I myself feel strain on my eyes looking on this colour scheme for long time and much better color theme is - you guess it - blueish
I guess best approach is to have support for basic themes (chose main and accent) color so people can change it if they feel uncomfortable

**Comment by heavymetalmixer** [comment]
I agree with your proposal, but you should change the title as it's confusing right now.

**Comment by Majoramari** [comment]
> I also not sure if It should have its own issue, but there are objective researches of how colour influences productivity and well-being and red color in particular is not optimal in that regard. I myself feel strain on my eyes looking on this colour scheme for long time and much better color theme is - you guess it - blueish I guess best approach is to have support for basic themes (chose main and accent) color so people can change it if they feel uncomfortable

I agree, I didn't like the default reddish color too.

**Comment by CaCTuCaTu4ECKuu** [comment]
I was never thinking about it because it just worked fine for me, but I explicitly checked and there is basic theme support and customization, it just slip my eyes, so all fine in that regard. But still by default is a bit bold choose.
Maybe ask for theme as first start like VS does?

**Comment by TVBrowntown** [comment]
Honestly, a red UI is not the play

**Comment by SkogiB** [comment]
Our changes did not nuke the existing themes system by the way! You can still select different themes in your settings menus
